Various updates (#11023)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/New-CsTeamsCallingPolicy.md

```

### -AllowCallRedirect

> Setting this parameter is enabling local call redirection for SIP devices connecting via the Microsoft Teams SIP gateway.

Reviewer: ```suggestion
Setting this parameter enables local call redirection for SIP devices connecting via the Microsoft Teams SIP gateway.

```

Valid options are:

> - Enabled: Enable the user to redirect an incoming call.
> - Disabled: User will not have such ability.

Reviewer: ```suggestion
- Enabled: Enables the user to redirect an incoming call.
- Disabled: The user cannot redirect an incoming call.

```
